shared/clients/draft-js/mentions-decorator/core.js

```diff
@@ -9,6 +9,28 @@ export type MentionComponentPropsType = {
   children: Node,
 };
 
+// 1. Get all the position of the mentions from start to finish
+//  "Hello @ale @ange @ale @ange" should return => [[6, 9], [11, 15], [17, 20], [22, 26]]
+// 2. With that array we can call the callback function.
+export const getMentionsPositionsFromMessage = (
+  message: string
+): Array<{ startPos: number, endPos: number }> => {
+  const mentionCoordinates = [];
+  const newMessage = message;
+
+  // $FlowFix
+  newMessage.replace(MENTIONS, (match, position) => {
```

Comment:
This is the better way I found to get the start and end position of the mentions